receiver/lokireceiver/README.md

```diff
@@ -4,11 +4,12 @@
 | Status        |           |
 | ------------- |-----------|
 | Stability     | [alpha]: logs   |
-| Distributions | [contrib], [sumo] |
+| Distributions | [contrib], [grafana], [sumo] |
```

Comment:
Same question as about the exporter

Reply:
Removed, for the same reason :) Great catch!

Reply:
Was it removed and the change not pushed?